### DISCUSSION TITLE
I get “An error occurred loading the update” when I try to install macOS Sonoma. What can I do to fix it?

### DISCUSSION THREAD

**Discussion by Jayman2000** [quickemu-project/quickemu#1262] [Quickemu]
Hello!

I’m currently working on contributing to [another open-source project](https://github.com/DescentDevelopers/Descent3). Specifically, I’m rewriting its build instructions to make them more clear and easier to follow. I’ve already tested out my new build instructions on Linux and Windows. All that’s left is to test them out on macOS.

I tried using [NixThePlanet](https://github.com/MatthewCroughan/NixThePlanet) to start a macOS VM and test out my new build instructions. NixThePlanet only supports macOS 13, and when I tried to install the latest version of Xcode, it told me that I needed macOS 14.0 or higher. I tried upgrading the NixThePlanet VM to macOS 14.5, but it gave me an error (I don’t remember what error specifically).

After that, I found quickemu and tried following [the instructions on the wiki](https://github.com/quickemu-project/quickemu/wiki/03-Create-macOS-virtual-machines) to install macOS 14. I got to the “Complete the installation as you normally would” step. After staying on this screen for a while:

![The installer says ‘macOS Sonoma will be installed on the disk “Main Disk”.’](https://github.com/quickemu-project/quickemu/assets/5579359/71b50351-0afd-4fdb-9f1a-3389edca2f13)

the installer would stop and tell me “An error occurred loading the update”:

![Once the error appears, the only option is to click “OK”.](https://github.com/quickemu-project/quickemu/assets/5579359/3c8e677c-5401-4dc9-92a6-d7ab1dd40caf)

 I was able to save a log by going to “Window > Installer Log”. You can find the log [here](https://gist.github.com/Jayman2000/ce9355dccab338f120f91149cc1c7d4c).

What can I do to fix this problem?

**Comment by tomchiverton** [comment]
I don't even get that far, any hints ?

https://github.com/quickemu-project/quickemu/issues/1265

**Reply by Jayman2000** [reply]
I tried to create a macOS Bug Sur VM, but I wasn’t able to reproduce that bug that you linked to. It looks like your host machine is running Ubuntu 22.04 on an Intel processor. My host machine runs NixOS 23.11 on an AMD processor. Maybe those two things have something to do with it, but I don’t really know.

**Comment by lj3954** [comment]
Regarding macOS issues, please try the latest version of quickemu from git. #1225 offers many fixes for both macOS hosts and guests, and it's not included in release 4.9.4.

**Reply by Jayman2000** [reply]
I tried version d78255b097b599e8ab3713cb61c4085cc45f5a95, and I got the same error.
